packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -556,9 +570,15 @@ mixin WidgetsBinding on BindingBase, SchedulerBinding, GestureBinding, RendererB
   /// Whether the Flutter engine has rasterized the first frame.
   ///
   /// {@macro flutter.frame_rasterized_vs_presented}
-  Future<void> get firstFrameRasterized => _firstFrameCompleter.future;
+  bool get firstFrameRasterized => _firstFrameCompleter.isCompleted;
 
-  /// Whether the first frame has finished rendering.
+  /// A future that completes when the Flutter engine has rasterized the first
+  /// frame.
+  ///
+  /// {@macro flutter.frame_rasterize`_vs_presented}
```

Comment:
This should probably have a "see also" section (see below for style) linking to firstFrameRasterized.

Reply:
Done.